config/features/flags.go

```diff
@@ -45,6 +45,10 @@ var (
 		Name:  "disable-peer-scorer",
 		Usage: "Disables experimental P2P peer scorer",
 	}
+	disableReorgLateBlocks = &cli.BoolFlag{
+		Name:  "disable-reorg-late-blocks",
```

Comment:
Does lighthouse have it enabled as default too? 

My initial feeling is we should have it disabled as default to be safe and then in subsequent releases. Revert it